src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx

```diff
@@ -34,9 +34,9 @@ type AmountFieldName = SendFormFields.FiatAmount | SendFormFields.CryptoAmount
 type UseSendDetailsReturnType = {
   balancesLoading: boolean
   fieldName: AmountFieldName
-  handleInputChange(inputValue: string): void
   handleNextClick(): void
   handleSendMax(): Promise<void>
+  inputHandler(inputValue: string): Promise<void>
```

Comment:
Why the name change here? I get the signature change but this is still handling input change

Reply:
I agree. I changed the name back to hanldeInputChange.